src/electos/ballotmaker/make_ballots.py

```diff
@@ -0,0 +1,18 @@
+"""Generate ballots from EDF data
+    Requires: 
+        EDF file (JSON format) edf_file: Path, 
+    Optional:
+        Output directory for generated PDF files
+        Settings file for ballot generation
+"""
+from pathlib import Path
+from electos.ballotmaker.constants import NO_ERRORS
+
+
+def make_ballots(
+    _edf: Path = None, _output_dir: Path = None, _settings: Path = None
+) -> int:
+    # is the EDF a JSON file?
+    # was a valid output directory provided?
+    # was a settings file provided?
```

Comment:
Useful stub comments 👍🏼

Reply:
Thanks!  I realized that "settings" is too generic so I'm switching over to the more descriptive "styles" because those settings are specifically for ballot layout issues (fonts, paper sizes, instructions, etc.)

Reply:
I also added some simple tests to see if a file path is passed in